spec/services/calculators/dependant_allowance_calculator_spec.rb

```diff
@@ -133,25 +133,25 @@ module Calculators
 
```

Comment:
I think we need two sets of tests here - one with a submission date of 5th April 2020 which should return the old values, and one with a submission date after the 5th april that should return the new dates.